pandas/core/frame.py

```diff
@@ -4818,7 +4818,8 @@ def aggregate(self, func, axis=0, *args, **kwargs):
 
     def apply(self, func, axis=0, broadcast=None, raw=False, reduce=None,
               result_type=None, args=(), **kwds):
-        """Applies function along an axis of the DataFrame.
+        """
+        Apply function along an axis of the DataFrame.
```

Comment:
Maybe "Apply a function along an axis of the DataFrame." (add "a" before "function").